Library/Formula/bro.rb

```diff
@@ -15,6 +15,7 @@ class Bro < Formula
   depends_on "swig" => :build
   depends_on "openssl"
   depends_on "geoip" => :recommended
+  conflicts_with "brotli", :because => "Both install a `bro` binary"
```

Comment:
Should be an empty line between the last `depends_on` and the first `conflicts_with`, Thanks!